powerlevel9k.zsh-theme

```diff
@@ -1505,6 +1517,22 @@ prompt_todo() {
   fi
 }
 
+# taskwarrior: show data from taskwarrior
+prompt_tw() {
+  if $(hash task 2>&-); then
+    today=$(task status:pending +DUETODAY count)
+    over=$(task status:pending +OVERDUE count)
+    pending=$(task status:pending count)
+    if [[ "$pending" = <-> ]]; then
```

Comment:
Does that mean that overdue tasks are pending as well? Otherwise you'll miss the overdue ones, if there are no pending ones.
This may be expressed by states as well. Probably if you want to change the color if there are overdue tasks..

Reply:
Yes, overdue tasks are pending as well. This means we could
simplify the code to `task +OVERDUE count`. Also there is
a better syntax for pending tasks: +PENDING.
The idea to use states to represent this information seems
really good, however I'm not sure how to go about implementing it.